airflow/models/xcom_arg.py

```diff
@@ -370,6 +370,13 @@ def resolve(self, context: Context, session: Session = NEW_SESSION) -> Any:
             return result
         if self.key == XCOM_RETURN_KEY:
             return None
+        if self.operator.multiple_outputs:  # type: ignore
```

Comment:
What is this ignoring?

Reply:
The operator typing is Operator which is a union of BaseOperator and MappedOperator but the type that get's there when using @task is _PythonDecoratedOperator. Might need to have a union of Operator and PythonDecoratedOperator as the type but I didn't look deeper
```
Run mypy for core..................................................................Failed
- hook id: mypy-core
- exit code: 1

WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
airflow/models/xcom_arg.py:373: error: Item "BaseOperator" of
"Union[BaseOperator, MappedOperator]" has no attribute "multiple_outputs" 
[union-attr]
            if self.operator.multiple_outputs:
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
airflow/models/xcom_arg.py:373: error: Item "MappedOperator" of
"Union[BaseOperator, MappedOperator]" has no attribute "multiple_outputs" 
[union-attr]
            if self.operator.multiple_outputs:
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
What do you think about having it in Operator here: https://github.com/apache/airflow/blob/fe7a1aa0962538a99f5e73dbd3890a6f210d22ed/airflow/models/operator.py#L27 but in a separate PR?

Reply:
Sounds like using `getattr(self.operator, "multiple_outputs", False)` would be fine here; if the operator does not support `multiple_outputs` the block does not matter anyway.

Reply:
Reviews are good! `getattr` didn't come to my mind at all. Thanks. Updated!